### PR TITLE
Feature/organizations and groups deletes

### PR DESCRIPTION
**Pull request by MrZyr0**

### CODE REVIEW

**Review by ChxGuillaume — APPROVED** [review]

File: src/services/group.service.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export class GroupService {
 
   getGroupsForOrganization = async (organizationId: string) =>
     this.groupRepository.findManyBy({
-      organization: new Types.ObjectId(organizationId),
+      organization: new mongoose.Types.ObjectId(organizationId),
```
Comment (MaxenceMottard) [inline src/services/group.service.ts:47]: unnecessary change but okay 😆

File: src/services/group.service.ts [hunk 2]
```diff
@@ -104,4 +104,26 @@ export class GroupService {
...
   };
+
+  removeUser = async (groupId: string, userId: string): Promise<void> => {
+    await this.groupRepository.Model.updateOne(
+      {
+        _id: groupId,
+      },
+      {
+        $pull: { users: userId },
+      },
```
Comment (MaxenceMottard) [inline src/services/group.service.ts:115]:
```suggestion
      { _id: groupId },
      { $pull: { users: userId },
```

File: src/services/group.service.ts [hunk 3]
```diff
@@ -104,4 +104,26 @@ export class GroupService {
...
+  };
+
+  removeAdmin = async (groupId: string, userId: string): Promise<void> => {
+    await this.groupRepository.Model.updateOne(
+      {
+        _id: groupId,
+      },
+      {
+        $pull: { admins: userId },
+      },
```
Comment (MaxenceMottard) [inline src/services/group.service.ts:126]:
```suggestion
      { _id: groupId },
      { $pull: { admins: userId } },
```

File: src/services/organization.service.ts [hunk 4]
```diff
@@ -108,4 +103,41 @@ export class OrganizationService {
...
+      },
+    );
+
+    await this.organizationRepository.Model.updateOne(
+      {
+        _id: organizationId,
+      },
+      {
+        $pull: { users: userId },
+      },
```
Comment (MaxenceMottard) [inline src/services/organization.service.ts:126]:
```suggestion
      { _id: organizationId ,
      { $pull: { users: userId } },
```

**Review by MaxenceMottard — APPROVED** [review]

File: src/services/group.service.ts (hunk 1 above)
Comment (MrZyr0) [reply src/services/group.service.ts:47]: It's more explicit :)
